core/src/commonMain/kotlin/com/saveourtool/processbuilder/Engine.kt

```diff
@@ -0,0 +1,17 @@
+
+
```

Comment:
why you added two lines here? and why diktat doesn't detect it

Reply:
Have no idea why it doesn't detect the problem. Diktat seems to be ok with this, while detekt doesn't seem to work. Will fix detekt in future PR

Reply:
Removed redundant lines